cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -33,18 +34,42 @@
 from cirq.testing.json import ModuleJsonTestSpec, spec_for
 
 REPO_ROOT = pathlib.Path(__file__).parent.parent.parent.parent
-TESTED_MODULES = [
-    'cirq_aqt',
-    'cirq_ionq',
-    'cirq_google',
-    'cirq.protocols',
-    'non_existent_should_be_fine',
-]
+
+
+@dataclasses.dataclass
+class _ModuleDeprecation:
+    old_name: str
+    deprecation_assertion: contextlib.AbstractContextManager
+
+
+# tested modules and their deprecation settings
+TESTED_MODULES: Dict[str, Optional[_ModuleDeprecation]] = {
+    'cirq_aqt': _ModuleDeprecation(
+        old_name="cirq.aqt",
+        deprecation_assertion=cirq.testing.assert_deprecated(
+            "cirq.aqt", deadline="v0.14", count=None
+        ),
+    ),
+    'cirq_ionq': _ModuleDeprecation(
+        old_name="cirq.ionq",
+        deprecation_assertion=cirq.testing.assert_deprecated(
+            "cirq.ionq", deadline="v0.14", count=None
+        ),
+    ),
+    'cirq_google': _ModuleDeprecation(
+        old_name="cirq.google",
+        deprecation_assertion=cirq.testing.assert_deprecated(
+            "cirq.google", deadline="v0.14", count=None
+        ),
+    ),
+    'cirq.protocols': None,
+    'non_existent_should_be_fine': None,
+}
 
 
 def _get_testspecs_for_modules():
     modules = []
-    for m in TESTED_MODULES:
+    for m in TESTED_MODULES.keys():
```

Comment:
Use the default iterator instead of calling `.keys()` on the dictionary.

Reply:
Why? I'm not using the dictionary part...

Reply:
Using default iterators, whenever supported, is more generic and avoids an unnecessary function call. 

https://google.github.io/styleguide/pyguide.html#28-default-iterators-and-operators